extensions/lifecycle/LifecycleQueuePopulator.js

```diff
@@ -96,28 +96,70 @@ class LifecycleQueuePopulator extends QueuePopulatorExtension {
         });
     }
 
+    _isBucketdEntry(entry) {
```

Comment:
It seems you are specifically looking for bucket change entries, so maybe rename to `isBucketEntry` (assuming you also remove the `isMongoEntry` helper)

Reply:
I believe the `isMongoEntry` (will rename to `isFileMDEntry`) is still needed to handle the `file` MD backend

Reply:
OK that works. But I believe the name should reflect that it's selecting bucket-update entries only, where the current name suggests it's filtering all entries coming from bucketd.

Reply:
What about: `isBucketUpdateFromBucketd()` and `isBucketUpdateFromFileMD`